src/main/java/com/psk/pyragas/ToolRent/entities/Order.java

```diff
@@ -6,30 +6,33 @@
 import javax.persistence.*;
 import java.io.Serializable;
 import java.math.BigDecimal;
-import java.util.Date;
+import java.time.LocalDateTime;
 
 @Entity
 @Getter
 @Setter
 @NamedQueries({
-        @NamedQuery(name = "Message.findAll", query = "select m from Message as m")
+        @NamedQuery(name = "Order.findAll", query = "select o from Order as o"),
+        @NamedQuery(name = "Order.findAllByProfile", query = "select o from Order as o where o.profile.id = :profileId" )
 })
 @Table(name = "order_table")
 public class Order implements Serializable {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    private BigDecimal price;
+    private BigDecimal rentPrice;
+    private BigDecimal operatorPrice;
 
     //TODO: think over naming, make enum for status
+    // Vykdomas, įvykdytas
     private String status;
 
     private String deliveryLocation;
 
-    private Date rentTimeStart;
+    private Long durationInHours;
 
-    private Date rentTimeEnd;
```

Comment:
I assume that this broke order creation.

Reply:
I don't think we need to add this property.
The thing that threw an exception is not orders.xhtml but detailed_advertisement.xhtml.
There this property is used but I deleted it because for me it seems much more logical to store rent start time and duration in database. There will be less errors, less checking of "is end date after start date". Also, it is much easier to display data about order in UI because you don't need to do complicated logic of calculating the interval of renting.
So I think we should delete this property from entity and edit detailed_advertisement.xhtml.

Reply:
Yeah, BA would probably ask us why is the interval of renting not calculated. And we would probably not implement that in time.

Reply:
Okay, I see that calendar is implemented in choosing time. Didn't notice it before.

Reply:
Probably the best way would be to leave the property as you did, and just make super simplified version of interval counting, just interval of hours. And in demo show just rent that is for 2 hours or something, in order not to accidentaly display "502 hours somewhere" 😀